src/js/view/utils/breakpoint.js

```diff
@@ -1,31 +1,28 @@
-define([
-    'utils/helpers'
-], function (utils) {
-    return function setBreakpoint(playerElement, playerWidth, playerHeight) {
-        var width = playerWidth;
-        var height = playerHeight;
+const utils = require('utils/helpers');
```

Comment:
`require('utils/dom');` dom utils is more specific than helpers which is really only meant to bundle utils into publicly available `jwplayer.utils` and to reduce imports in large modules.